vendor/build_module.sh

```diff
@@ -0,0 +1,424 @@
+#!/usr/bin/env bash
+#
+# build_module.sh (c) NGINX, Inc. [v0.12 30-Aug-2017] Liam Crilly <liam.crilly@nginx.com>
+# This script is downloaded from http://hg.nginx.org/pkg-oss/raw-file/default/build_module.sh
```

Comment:
can you pull this out above the block comment? It's getting lost in the other text